### DISCUSSION TITLE
[Question]: Viewing/identifying historic motion detection (not event/object detection) while recording continuously

### DISCUSSION THREAD

**Discussion by B0ycee** [blakeblackshear/frigate#11004] [Ask A Question]
### What is your question:

Just wanted to check I’m not missing something as it seems that when having continuous recording there is no method to show/view historic motion detection (not event/object detection)?

I guess the first prize solution would be some kind of histogram overlay with the frequency of motion detection shown below the recording seek bar, allowing the user to easily seek to “busy” motion periods, but alternatively just being able to play the motion segments in isolation as (I imagine) would be recorded by the motion only record mode?

Only thing I can think would be to record the same camera under another name in parallel with record mode set to motion?  That won’t be great for storage/efficiency.

(I know there are requests/discussions around separate retention times for continuous/motion/event etc, but this is specifically around reviewing the video with motion detection.)

**Comment by NickM-27** [comment]
In the current version of frigate you are correct, however frigate is currently undergoing a UI rewrite and a motion review screen has already been implemented including a timeline showing motion levels
